src/extension/csv-with-metadata-header/src/main/java/au/org/emii/geoserver/wfs/response/CSVWithMetadataHeaderOutputFormat.java

```diff
@@ -175,8 +175,10 @@ private void writeMetadata(FeatureCollectionResponse featureCollection,
             rs = stmt.executeQuery();
 
             while (rs.next()) {
-                w.write(rs.getString(1));
-                w.newLine();
+                if (rs.getString(1) != null) {
```

Comment:
Could store the result of getString to avoid the duplicate getString method call?
e.g.

```java
            while (rs.next()) {
                String value = rs.getString(1);
                if (value != null) {
                    w.write(value);
                    w.newLine();
                }
}
```

Reply:
Yeah, a bit cleaner. Change made.